tests/test_data/package-files/7.5.0/README.rst

```diff
@@ -56,6 +56,10 @@ Issues with plugins and modules in the Ansible package should be reported
 on the individual collection's issue tracker.
 Issues with ``ansible-core`` should be reported on
 the `ansible-core issue tracker <https://github.com/ansible/ansible/issues/>`_.
+Issues with the ``ansible`` package build process or serious bugs or
+vulnerabilities in a collection that are not addressed after opening an issue
+in the collection's issue tracker should be reported on
+`ansible-build-data's issue tracker <https://github.com/ansible-community/ansible-build-data/issues>`_.
```

Comment:
Should we mention the README here? I'm thinking of something like

```
For a more detailed overview, refer to the `ansible-build-data's README <https://github.com/ansible-community/ansible-build-data/blob/main/README.md>`_.
```

I'm not sure, so just commenting and not requesting a change.

Reply:
Sounds like a good idea, especially if we add https://github.com/ansible-community/ansible-build-data/pull/322.

Reply:
I like the suggestion, but I'm a bit hesitant to make this section longer. I'm open to adding that after https://github.com/ansible-community/ansible-build-data/pull/322 is merged if others feel strongly.

Reply:
why edit anything in 7.x? it's eol and no changes will happen.

Reply:
I think I know what you mean. But why should we say at all what kind of issues we accept in ansible-build-data here? Can't we _just_ refer to the README? Something like:

> If you think the above issue trackers are the wrong place for your issue, there is also the `ansible-build-data's issue tracker <https://github.com/ansible-community/ansible-build-data/issues>`_. Please refer to the `ansible-build-data's README <https://github.com/ansible-community/ansible-build-data/blob/main/README.md>` for more details.

This lacks the information about "serious bugs or vulnerabilities in a collection that are not addressed after opening an issue in the collection's issue tracker", though... but I think we should mention the README somehow here. We have a lot more details about acceptable issues there, far more than we could or should go into here.

Reply:
@samccann this is a fixure for unit tests. If any older verison is rebuilt, the latest README.rst template has always been used. Simply ignore the filename. (The comment should have been for src/antsibull/data/ansible-readme.rst anyway, since that's the source of truth.)

Reply:
Now https://github.com/ansible-community/ansible-build-data/pull/322has been merged :)